pyaerocom/trends_helpers.py

```diff
@@ -232,7 +232,11 @@ def _get_yearly(data, seas, start_yr):
             val = np.nan
 
         elif seas == "all":
-            seasons = _get_unique_seasons(subset.index)
+            d = subset.index
+            valid_mask = ~np.isnan(subset.values)
+            # seasons = _get_unique_seasons(d) # original (EMEP raport)
+            seasons = _get_unique_seasons(d[valid_mask])
+
```

Comment:
This is a new addition which I'm not sure was added, but which makes sense. I need to look at this